### PR TITLE
fix description --oidc-issuer-url.md

### PR DESCRIPTION
**Pull request by nnlkcncff**
> https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens
> This is typically the provider's discovery URL without a path, for example "https://accounts.google.com" or "https://login.salesforce.com". This URL should point to the level below .well-known/openid-configuration.
> If the discovery URL is `https://accounts.google.com/.well-known/openid-configuration`, the value should be https://accounts.google.com.

It seems that there is a small mistake here — instead of the word “below” there should be the word “above”:

```plaintext
https://accounts.google.com/           <= above
└─ .well-known/openid-configuration    <= below
```

If I'm not mistaken consider to change the phrase to "This URL should point to the level above .well-known/openid-configuration."

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
I think ”below” is correct and haven't yet seen evidence to the contrary.

**Comment by nnlkcncff** [comment]
> https://datatracker.ietf.org/doc/html/rfc3986#section-1.2.3
>
> The path segments "." and "..", also known as dot-segments, are
>  defined for relative reference within the path name hierarchy.  They
>  are intended for use at the beginning of a relative-path reference
>  ([Section 4.2](https://datatracker.ietf.org/doc/html/rfc3986#section-4.2)) to indicate relative position within the hierarchical
>  tree of names.  This is similar to their role within some operating
>  systems' file directory structures to indicate the current directory
>  and parent directory, respectively.

So URI path is similar to hierarchical file system path.

There is some explanations on [Wikipedia](https://en.wikipedia.org/wiki/Hierarchical_file_system), although without declaring terms such as "lower tree branch", "higher tree branch", but there is a term "top" which is referring on he root of tree.
> In a hierarchical file system, directories contain information about both [files](https://en.wikipedia.org/wiki/Computer_file) and other directories, called subdirectories which, in turn, can point to other subdirectories, and so on.[[2]](https://en.wikipedia.org/wiki/Hierarchical_file_system#cite_note-Infoworld-2) This is organized as a [tree structure](https://en.wikipedia.org/wiki/Tree_structure), or hierarchy, generally portrayed with the root at the top.

So I would say that "top" word is closer to "above" than to "below".

```plaintext
/              |    https://accounts.google.com
└─ usr         |    └─ .well-known
   └─ local    |       └─ openid-configuration
```

So `/usr` is above `/usr/local` just like `https://accounts.google.com/` is above `https://accounts.google.com/.well-known/openid-configuration`, don't you think so?

The same is true with the PCI tree `lspci -btv`, and with the disk partition tree `lsblk` — and so with each hierarchical data structure.

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -312,7 +312,7 @@ To enable the plugin, configure the following flags on the API server:
 
 | Parameter | Description | Example | Required |
 | --------- | ----------- | ------- | ------- |
-| `--oidc-issuer-url` | URL of the provider which allows the API server to discover public signing keys. Only URLs which use the `https://` scheme are accepted.  This is typically the provider's discovery URL without a path, for example "https://accounts.google.com" or "https://login.salesforce.com".  This URL should point to the level below .well-known/openid-configuration | If the discovery URL is `https://accounts.google.com/.well-known/openid-configuration`, the value should be `https://accounts.google.com` | Yes |
+| `--oidc-issuer-url` | URL of the provider which allows the API server to discover public signing keys. Only URLs which use the `https://` scheme are accepted.  This is typically the provider's discovery URL without a path, for example "https://accounts.google.com" or "https://login.salesforce.com".  This URL should point to the level above .well-known/openid-configuration | If the discovery URL is `https://accounts.google.com/.well-known/openid-configuration`, the value should be `https://accounts.google.com` | Yes |
```
Comment (sftim) [inline content/en/docs/reference/access-authn-authz/authentication.md:315]: I think we can take the question of “above“ or ”below” off the table:
```suggestion
| `--oidc-issuer-url` | URL of the provider that allows the API server to discover public signing keys. Only URLs that use the `https://` scheme are accepted.  This is typically the provider's discovery URL, changed to have an empty path | If the issuer's OIDC discovery URL is `https://accounts.provider.example/.well-known/openid-configuration`, the value should be `https://accounts.provider.example` | Yes |
```

**Review by sftim — COMMENTED** [review]
“above” makes more sense; however, it also feels weird to talk about “the level above” something (`.well-known) when that something is always relative to the top of the site.

How about this wording?

File: content/en/docs/reference/access-authn-authz/authentication.md (hunk 1 above)
Comment (nnlkcncff) [reply content/en/docs/reference/access-authn-authz/authentication.md:315]: @sftim, your proposal is better than what is currently in the documentation, because without position words the ambiguity disappears.

Comment (tengqm) [reply content/en/docs/reference/access-authn-authz/authentication.md:315]: @nnlkcncff Are you planning to revise based on the comments provided by @sftim ?

Comment (nnlkcncff) [reply content/en/docs/reference/access-authn-authz/authentication.md]: @tengqm, sorry, I thought I've included the suggestion of @sftim and there was no longer any need for other actions. I've added his changes.

**Comment by onlydole** [comment]
Thank you for your work on this, @nnlkcncff! 

/approve
